### DISCUSSION TITLE
微信更新3.9.2.23后防撤回补丁失败

### DISCUSSION THREAD

**Discussion by yixinroadliangjie** [huiyadanli/RevokeMsgPatcher#537] [General]
微信更新3.9.2.23后防撤回补丁失败

**Comment by HunnyOvO** [comment]
+1

**Comment by nameIsLove** [comment]
大佬们 视频都不能看了

**Comment by chen12345jin** [comment]
防撤回3.9之后不能用了

**Reply by yixinroadliangjie** [reply]
可以用，我公司的电脑是3.9.0.XX版本的，可以正常使用。

**Comment by 1005772638** [comment]
+1

**Comment by yixinroadliangjie** [comment]
可以自行到微信官网下载3.9.0.28版本，重新降级安装以后，再打一次防撤回补丁，登陆以后再在设置里面关掉自动更新就好。问题解决了

**Reply by LiHangzls** [reply]
麻烦问您一下，降级安装的时候有提示覆盖，如果安装覆盖了，那么原来的历史聊天记录还在吗？谢谢了

**Reply by yixinroadliangjie** [reply]
你好，我就是降级安装，原来的聊天记录还在，問題不大

**Reply by LiHangzls** [reply]
有安装包吗，官网我没找到历史版本，或者有下载的链接吗，谢谢啦

**Reply by LiHangzls** [reply]
我先去试试豌豆荚，用你的3.9.0.28版本

**Reply by LiHangzls** [reply]
垃圾豌豆荚，百度太难搜了。我找到了一个3.9.2的版本，现在降版本了，也可以防撤回，谢谢！

**Reply by yixinroadliangjie** [reply]
你好，朋友，亲测3.9.5.81版本也可以防撤回，再新的就没测试啦

**Comment by kariuswong** [comment]
然后发现其实防撤回也不是那么必要，看开了就好

**Reply by yixinroadliangjie** [reply]
对我来说还是挺有用的，我很感谢作者开发了这个小玩意，如果可以的话，我想给他买个猪脚饭
